### PR TITLE
New strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -391,27 +391,27 @@
     <string name="title_behaviour">Behaviour</string>
     <string name="root_mode">Root mode</string>
     <string name="root_mode_summary">Used to hibernate apps</string>
-    <string name="root_mode_error">Unable to gain root privileges</string>
-    <string name="behaviour_link_close_keyboard_back_button">Close keyboard same as back button</string>
-    <string name="keyboard_suggestions">Enable autocomplete/suggestions</string>
+    <string name="root_mode_error">Could not get root access</string>
+    <string name="behaviour_link_close_keyboard_back_button">Close keyboard with «Back» button</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:395]: Not sure here

Comment (TBog) [reply app/src/main/res/values/strings.xml:395]: When the keyboard reports the action `android.R.id.closeButton` and `linkCloseKeyboardToBackButton` returns true the code will call `Behaviour.onBackPressed`.

All `Behaviour.onBackPressed` does is clear the search bar (and result list) then executa the action defined in the `Behaviour` -> `Desktop mode ...` -> `Back button` preference

Comment (TBog) [reply app/src/main/res/values/strings.xml:395]: "behaviour_link_close_keyboard_back_button" should be something like:
_When you close the keyboard the action executed when you would have pressed the back button will also execute._
But it's kind of long :stuck_out_tongue:

Comment (comradekingu) [reply app/src/main/res/values/strings.xml:395]:
```suggestion
    <string name="behaviour_link_close_keyboard_back_button">Dual-action «Back» button</string>
```
?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -391,27 +391,27 @@
     <string name="title_behaviour">Behaviour</string>
     <string name="root_mode">Root mode</string>
     <string name="root_mode_summary">Used to hibernate apps</string>
-    <string name="root_mode_error">Unable to gain root privileges</string>
-    <string name="behaviour_link_close_keyboard_back_button">Close keyboard same as back button</string>
-    <string name="keyboard_suggestions">Enable autocomplete/suggestions</string>
+    <string name="root_mode_error">Could not get root access</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:394]:
```suggestion
    <string name="root_mode_error">Could not gain root access</string>
```

Comment (TBog) [reply app/src/main/res/values/strings.xml:394]: `gain` sounds ok.

Comment (TBog) [reply app/src/main/res/values/strings.xml:394]: But this is wrong:
```
<string name="behaviour_link_close_keyboard_back_button">Close keyboard with «Back» button</string>
```

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:395]:
```suggestion
    <string name="behaviour_link_close_keyboard_back_button">\"Back\" action when closing keyboard</string>
```

**Pull request merged**